Dockerfile

```diff
@@ -1,25 +1,35 @@
-FROM node:18-alpine
+# Stage 1: Build Stage.
+FROM node:18-alpine AS build
 
-# Set working directory
-WORKDIR /app/docs
+# Set working directory.
+WORKDIR /app
 
-# Copy package.json and package-lock.json to the container
+# Copy package.json and package-lock.json to the container.
 COPY package.json package-lock.json ./
 
-# Install dependencies
-RUN npm install
+# Install dependencies and browser-sync globally.
+RUN npm install && npm install -g browser-sync
 
-# Copy current directory to the container
+# Copy the rest of the application files.
 COPY . .
 
-# Set environment variable
-ENV NODE_ENV=development
+# Stage 2: Development Stage.
+FROM alpine:latest
```

Comment:
I understand that the final image is based on `alpine:latest`, and `node:18-alpine` was used before. Does it matter?